Jenkinsfile

```
@@ -1,6 +1,12 @@
 #!/usr/bin/env groovy
 
-buildPlugin(useContainerAgent: true, configurations: [
-  [ platform: "windows", jdk: "11" ],
-  [ platform: "linux", jdk: "11" ]
-])
\ No newline at end of file
+buildPlugin(useContainerAgent: true,
+        configurations: [
+                [platform: 'linux', jdk: '11'],
+                [platform: 'windows', jdk: '8'],
```

I've recently removed the compilation with JDK 8 it is not needed

```suggestion
                [platform: 'windows', jdk: '11'],
```

---

As you see fit.

Applied